File: ProgrammersGuide/WorkingWithProjects/CreatingEmptyProjectFile/CreatingEmptyProjectFileExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammersGuide.WorkingWithProjects.CreatingEmptyProjectFile
{
    [TestClass, TestFixture]
    public class CreatingEmptyProjectFileExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void CreateEmptyProjectAndSaveXMLFile()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CreatingEmptyProjectFile/CreateEmptyProjectAndSaveXMLFile");

            CreateEmptyProjectAndSaveXMLFileExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void CreateEmptyProjectSavetoStream()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CreatingEmptyProjectFile/CreateEmptyProjectSavetoStream");

            CreateEmptyProjectSavetoStreamExample.Program.Main();
        }

    }
}

File: Examples/ProgrammersGuide/WorkingWithProjects/CreatingEmptyProjectFile/CreatingEmptyProjectFileExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammersGuide.WorkingWithProjects.CreatingEmptyProjectFile
	<TestClass, TestFixture> _
	Public Class CreatingEmptyProjectFileExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub CreateEmptyProjectAndSaveXMLFile()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CreatingEmptyProjectFile/CreateEmptyProjectAndSaveXMLFile")
	
	            CreateEmptyProjectAndSaveXMLFileExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub CreateEmptyProjectSavetoStream()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/CreatingEmptyProjectFile/CreateEmptyProjectSavetoStream")
	
	            CreateEmptyProjectSavetoStreamExample.Program.Main()
        End Sub

	End Class
End Namespace